libs/aws/langchain_aws/agents/base.py

```diff
@@ -480,6 +490,7 @@ def invoke(
                 intermediate_steps: The intermediate steps that are used to provide RoC
                     invocation details
                 enable_trace: Boolean flag to enable trace when invoke bedrock agent
```

Comment:
in langchain, the approach is typically set the debug and verbose flag globally.

but comparing the BedrockAgentsRunnable with Agents CRUD API and invoke with Bedrock Agents Runtime, the enable_trace is a field on runtime API which typically translates to invoke method on LangChain Runnable.

i went with the BedrockAgents construct so that it can direct map to the invoke_agent. let me know if we should do it the other way.

Reply:
as per our conversation -- moved the flag to the Runnable.